beater/config/config_test.go

```diff
@@ -404,32 +404,6 @@ func TestTLSSettings(t *testing.T) {
 		}
 	})
 
-	t.Run("VerificationMode", func(t *testing.T) {
```

Comment:
I'm on board with removing docs/references to the config, as it's only meant for testing purposes, but I think we should keep the tests since the config will still take effect if someone sets it IIANM.

Reply:
Disregard, I didn't look closely enough. As you said, it's not relevant for `apm-server.ssl.*`